CppCoreGuidelines.md

```diff
@@ -12663,11 +12663,11 @@ If you really need to break out a loop, a `break` is typically better than alter
 
 ???
 
-### <a name="Res-break"></a>ES.78: Always end a non-empty `case` with a `break`
+### <a name="Res-break"></a>ES.78: Don't rely on implicit fallthrough in `switch` statements
```

Comment:
Why is the rule changed? What is the benefit?
- The new rule contradicts with the second example, the old rule not.
- The old rule is now added in the reason section, that makes this rules longer than before.

Reply:
adding the exit by return is important; also, I guess, throw.